modules/customizing-and-automating-your-compliance-scans.adoc

```diff
@@ -1,24 +1,27 @@
 // Module included in the following assemblies:
 //
-// * manage-compliance/managing-compliance-20.adoc
+// * operating/managing-compliance.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="customizing-and-automating-your-compliance-scans_{context}"]
 = Customizing and automating your compliance scans
 
 By creating a compliance scan schedule, you can customize and automate your compliance scans to align with your operational requirements.
 
+:FeatureName: Configuring scan schedules
+include::snippets/technology-preview.adoc[]
+
```

Comment:
Would it make sense to include a note on the limitation of scanning the same profile and the same cluster here? Something along the lines of:

"Note: Only one schedule can scan the same profile on the same cluster. This means you cannot create multiple scan schedules for the same profile on a single cluster"